### PR TITLE
No astropy helpers

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -47,17 +55,38 @@ show-response = 1
...
 
 [pycodestyle]
-exclude = extern,sphinx,*parsetab.py,astropy_helpers,ah_bootstrap.py,conftest.py,docs/conf.py,setup.py
+select = E101,W191,W291,W292,W293,W391,E111,E112,E113,E901,E902
+exclude = extern,sphinx,*parsetab.py
 max_line_length = 100
+
+[entry_points]
+
+ # astropy-package-template-example = packagename.example_mod:main
```
Comment (nabobalis) [inline setup.cfg]:
```suggestion
```

File: roentgen/conftest.py [hunk 2]
```diff
@@ -1,49 +1,7 @@
...
-
-# This is to figure out the package version, rather than
-# using Astropy's
-from .version import version, astropy_helpers_version
+import pytest
+import astropy.units as u
 
 packagename = os.path.basename(os.path.dirname(__file__))
-TESTED_VERSIONS[packagename] = version
-TESTED_VERSIONS['astropy_helpers'] = astropy_helpers_version
```
Comment (nabobalis) [inline roentgen/conftest.py]: I don't think you need tihs file.

File: roentgen/_astropy_init.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+# probably don't need this anymore!
```
Comment (nabobalis) [inline roentgen/_astropy_init.py]: Yeah if this is still in the base template, we should remove it.

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    jobs:
+      - egg-info-37
+
+  pycodestyle:
+    jobs:
+      - pycodestyle
+
+  test-documentation:
+    jobs:
+      - html-docs
```
Comment (nabobalis) [inline .circleci/config.yml]:
```suggestion
      - html-docs

  tests:
    jobs:
      - tests 

```
Should make the final test run on circle.

File: roentgen/_astropy_init.py (hunk 3 above)
Comment (ehsteve) [reply roentgen/_astropy_init.py]: I did not use the new template. I was trying to migrate myself.

Comment (nabobalis) [reply roentgen/_astropy_init.py]: Oh in that case, we can remove this but will need some code in setup.py and in another file.

File: roentgen/conftest.py (hunk 2 above)
Comment (ehsteve) [reply roentgen/conftest.py]: You mean I should delete these two lines? Not sure what file you are referring to.

Comment (nabobalis) [reply roentgen/conftest.py]: This file is `roentgen/conftest.py`

File: roentgen/_astropy_init.py (hunk 3 above)
Comment (ehsteve) [reply roentgen/_astropy_init.py]: What do you mean given that this file is not used at all and everything seems to work now. What code would need to be added in other places?

Comment (nabobalis) [reply roentgen/_astropy_init.py]: I think it should be imported by the ``__init__.py`` from what i recall from the old template.

File: roentgen/conftest.py (hunk 2 above)
Comment (ehsteve) [reply roentgen/conftest.py:49]: Removed.

**Pull request merged**